lib/modules/pipeline/webpack.config.js

```diff
@@ -8,25 +8,25 @@ const embarkPath = process.env.EMBARK_PATH;
 const dappNodeModules = path.join(dappPath, 'node_modules');
 const embarkNodeModules = path.join(embarkPath, 'node_modules');
 
-function customRequire(mod) {
-  return require(customRequire.resolve(mod));
+function requireFromEmbark(mod) {
+  return require(requireFromEmbark.resolve(mod));
 }
 
-customRequire.resolve = function (mod) {
+requireFromEmbark.resolve = function (mod) {
   return require.resolve(
     mod,
-    {paths: [dappNodeModules, embarkNodeModules]}
+    {paths: [embarkNodeModules]}
   );
 };
 
 // some packages, plugins, and presets referenced/required in this webpack
 // config are deps of embark and will effectively be transitive dapp deps
 // unless specified in the dapp's own package.json
 
-const cloneDeep = customRequire('lodash.clonedeep');
-// const CompressionPlugin = customRequire('compression-webpack-plugin');
-const glob = customRequire('glob');
-const HardSourceWebpackPlugin = customRequire('hard-source-webpack-plugin');
+const cloneDeep = requireFromEmbark('lodash.clonedeep');
+// const CompressionPlugin = requireFromEmbark('compression-webpack-plugin');
```

Comment:
Do we still need that comment?

Reply:
If we remove it then the linter will complain that the `CompressionPlugin` variable is never used. So it either needs to be commented, or we can completely remove it and a related commented line in the production-config section (toward the bottom).

Reply:
Yeah, I was mostly asking if we can remove the line. Will it be needed by a user?

Reply:
I thought that it could be helpful if a user ejected the config and was studying it; also, at the time I included it, I thought we might move more quickly to integrating compression with our upload command. At this point, though, it's probably better to remove it.

Reply:
Done.